hamocc/mo_apply_oafx.F90

```diff
@@ -3,107 +3,75 @@
 
 
 module mo_apply_oafx
-!******************************************************************************
-!
-!   J.Schwinger             *NORCE Climate, Bergen*             2021-11-15
-!
-! Modified
-! --------
-!
-! Purpose
-! -------
-!  -Routines for applying ocean alkalinization
-!
-!
-! Description:
-! ------------
-!
-!  -subroutine alkalinization
-!     Apply alkalinization to the top-most model layer.
-!
-!
-!******************************************************************************
-  implicit none
 
+  !******************************************************************************
+  ! Routines for applying ocean alkalinization
+  !
+  ! J.Schwinger             *NORCE Climate, Bergen*             2021-11-15
+  !******************************************************************************
+
+  implicit none
   private
-  public :: apply_oafx
 
-!******************************************************************************
-contains
+  public :: apply_oafx ! Apply alkalinization to the top-most model layer.
 
+contains
 
+  subroutine apply_oafx(kpie,kpje,kpke,pddpo,omask,oafx)
 
-subroutine apply_oafx(kpie,kpje,kpke,pddpo,omask,oafx)
-!******************************************************************************
-!
-!     J. Schwinger            *NORCE Climate, Bergen*     2021-11-15
-!
-! Purpose
-! -------
-!  -apply alkalinization to the top-most model layer.
-!
-! Changes: 
-! --------
-!
-!
-! Parameter list:
-! ---------------
-!  *INTEGER*   *kpie*    - 1st dimension of model grid.
-!  *INTEGER*   *kpje*    - 2nd dimension of model grid.
-!  *REAL*      *pddpo*   - size of grid cell (depth) [m].
-!  *REAL*      *omask*   - land/ocean mask (1=ocean)
-!  *REAL*      *oafx*    - alkalinization field to apply [kmol m-2 yr-1]
-!
-!******************************************************************************
-  use mo_control_bgc, only: dtb,do_oalk
-  use mo_param1_bgc,  only: ialkali
-  use mo_carbch,      only: ocetra,oalkflx,OmegaA
-  use mo_read_oafx,   only: thrh_omegaa
+    !******************************************************************************
+    ! Apply alkalinization to the top-most model layer.
+    !
+    ! J. Schwinger            *NORCE Climate, Bergen*     2021-11-15
+    !******************************************************************************
 
-  implicit none
+    use mo_control_bgc, only: dtb,do_oalk
+    use mo_param1_bgc,  only: ialkali
+    use mo_carbch,      only: ocetra,oalkflx,OmegaA
+    use mo_read_oafx,   only: thrh_omegaa
 
-  integer, intent(in) :: kpie,kpje,kpke
-  real,    intent(in) :: pddpo(kpie,kpje,kpke)
-  real,    intent(in) :: omask(kpie,kpje)
-  real,    intent(in) :: oafx(kpie,kpje)
+    ! Arguments
+    integer, intent(in) :: kpie                       ! 1st dimension of model grid.
+    integer, intent(in) :: kpje                       ! 2nd dimension of model grid.
+    integer, intent(in) :: kpke                       ! size of grid cell (depth) [m].
```

Comment:
comments need update: kpke: vertical dimension, pddpo: size of grid cell, omask: loand/ocean mask, oafx: alkalinization field to apply

Reply:
Done. Thanks for catching this.